.github/workflows/publish.yml

```diff
@@ -114,85 +121,97 @@ jobs:
       S3_BUCKET: ${{ secrets.S3_BUCKET }}
 
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout
+        uses: actions/checkout@v2.4.0
         with:
           fetch-depth: 0
 
-      - name: "Setup"
-        run: brew install grep
+      - name: Set up Java
+        uses: actions/setup-java@v2.4.0
+        with:
+          distribution: 'zulu'
+          java-version: 11
+
+      - name: Download Arrow version file
+        uses: actions/download-artifact@v2
+        with:
+          name: arrow-version
```

Comment:
This is not required with the latest change, no?

Reply:
True! Good spot!